src/Store/reducers/allDataTableReducer.js

```diff
@@ -1,13 +1,15 @@
 const defaultState = {
   valuesEdited: false,
-  editModalOpen: false,
-  editModalData: {},
+  editLocationModalOpen: false,
```

Comment:
valuesEdited, editLocationModalData and editLocationModalOpen actually aren't ever set in this file and should be removed from defaultState